### DISCUSSION TITLE
Refiner not showing all values

### DISCUSSION THREAD

**Discussion by dignus93** [microsoft-search/pnp-modern-search#3612] [Q&A] [answered]
Hello folks!

I am using the latest release 4.10.2.0.
I am using the webpart to filter on a SharePoint list.
I have a RefinableString mapped to a crawled properties that is a SharePoint choice column with 2700 elements.

I am using this Refinable as filter in the "PnP - Search Filters" webpart. It works pretty fine but it is not showing all the elements.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/28197473/ee97a1d1-0d33-4125-aa5b-c100621baece)

I already set the # of values to 3000 in the Edit Filters section of the "PnP - Search Filters" webpart
![image](https://github.com/microsoft-search/pnp-modern-search/assets/28197473/9f933edc-3544-4335-bfd3-d7bff8c2fbef)

Any idea or a workaround to solve this issue?

**Comment by wobba** [comment, marked as answer]
We cannot control the number of items returned from the API, and no documentation on what the max limit is. I understand the scenario as you want to allow filtering for end-users, but using search for this will probably not work for this amount of values.
